### PR TITLE
 Refactor StreamK and SplitK benchmarks

### PR DESCRIPTION
**Pull request by LiyangLingIntel**

### CODE REVIEW

File: .github/workflows/triton-benchmarks.yml [hunk 1]
```diff
@@ -324,6 +324,10 @@ jobs:
       - name: Run triton gemm kernel benchmark
         run: |
           cd benchmarks/triton_kernels_benchmark
+          # Dry run to validate no functionality regression for splitk and streamk in defaul path
```
Comment (whitneywhtsang) [inline .github/workflows/triton-benchmarks.yml:327]:
```suggestion
          # Dry run to validate no functionality regression for splitk and streamk in default path
```

Comment (LiyangLingIntel) [reply .github/workflows/triton-benchmarks.yml]: Updated

**Comment by ESI-SYD** [comment]
CI testing: https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10137508185

**Comment by whitneywhtsang** [comment]
> CI testing: https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10137508185

FYI @chengjunlu The failures reported in CI are due to enabling large 2d block load a74da7d7c0a862940a9f6bcf21bfa5f55f608085.

**Comment by chengjunlu** [comment]
> > CI testing: https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10137508185
> 
> FYI @chengjunlu The failures reported in CI are due to enabling large 2d block load [a74da7d](https://github.com/intel/intel-xpu-backend-for-triton/commit/a74da7d7c0a862940a9f6bcf21bfa5f55f608085).

I updated the information to the issue https://github.com/intel/intel-xpu-backend-for-triton/issues/1716.

**Review by whitneywhtsang — APPROVED** [review]

**Comment by whitneywhtsang** [comment]
Since the benchmarks are failing cannot be added to CI, repurposed this PR to refactor the benchmarks.

**Pull request merged**